contracts/root/stakeManager/IStakeManager.sol

```diff
@@ -0,0 +1,6 @@
+pragma solidity ^0.5.5;
+
+interface IStakeManager {
```

Comment:
we already have StakeManager interface which follows EIP 900, we can include `checkSignatures ` in it.